styleguide-app/assets/reset.css

```diff
@@ -1,48 +0,0 @@
-/* http://meyerweb.com/eric/tools/css/reset/
-   v2.0 | 20110126
-   License: none (public domain)
-*/
```

Comment:
it looks like this didn't get ported to the CSS. Was it unnecessary?

Reply:
We don't use this reset on the monolith, and while we do some resetting/have inheritance issues, I don't think it's reasonable to rely on this code. You know? Like, we know we don't use it on the monolith, and we want our widgets to stand alone..